beast/physicsmodel/stars/stellib.py

```diff
@@ -1685,7 +1685,7 @@ class BTSettl(Stellib):
       http://adsabs.harvard.edu/abs/2016sf2a.conf..223A
       http://adsabs.harvard.edu/abs/2012RSPTA.370.2765A
 
-    Files available at: https://phoenix.ens-lyon.fr/Grids/BT-Settl/
+    Files used to be available at: (https)phoenix.ens-lyon.fr/Grids/BT-Settl/
```

Comment:
I can't find them.  And since this link is checked in as part of the online CI tests, it fails.  Hence keeping the info, but making it something that doesn't parse as a URL.